aip/auth/4122.md

```diff
@@ -0,0 +1,349 @@
+---
+id: 4122
+scope: auth
+state: reviewing
+created: 2020-12-15
+---
+
+# Service Account Impersonation
+
+A service account or a user identity in GCP can impersonate another service 
+account, and request short-lived credentials on behalf of that service account.
+
+The request can be direct or delegated.
```

Comment:
Does this mean the request for credentials? It might be worth italicising "direct" and "delegated" to indicate that these are terms defined below - or expand the sentence. (When I initially read this sentence I got very confused by it.)